samples/affinidi-nextjs-nextauthjs/src/components/NavBar.tsx

```diff
@@ -59,6 +59,13 @@ const NavBar: FC = () => {
           >
             Issue Credentials
           </Link>
+          <Link
+            href="/credential-revocation"
+            className="py-4 font-medium transition-colors hover:text-blue-500"
+            prefetch={false}
+          >
+            Revoke Credentials
+          </Link>
```

Comment:
I see that multiple changes are done directly on the `samples/affinidi-nextjs-nextauthjs` directory. This is an autogenerated folder. Please make sure to add all of your changes to the `generator/nextjs/template` directory and run `npm run generate:nextjs` from the root. Working directly on the sample apps is usually a bad idea as you risk loosing your changes if you run the generator. See [CONTRIBUTING.md](https://github.com/affinidi/reference-app-affinidi-vault/blob/main/CONTRIBUTING.md) for more info.